server/src/logic/utils/authUtils.ts

```diff
@@ -0,0 +1,42 @@
+import * as jwt from "jsonwebtoken";
+import { Router, Request, Response, NextFunction } from "express";
+import { Config } from "./../../config";
+
+
+export class AuthUtils {
+
+  static checkState (req: any, res: Response, next: NextFunction) {
```

Comment:
Yes, for instance I injected the validator into the recipe routes. But I don't want to force you to use my approach, I just think this would be good idea and it is also often suggested as good practice.